keyboards/kbdfans/kbd75/keymaps/tucznak/config.h

```diff
@@ -0,0 +1,46 @@
+#ifndef CONFIG_USER_H
```

Comment:
This file should be changed to use the `#pragma once` include guard.

```suggestion
#pragma once
```